test/unit/BPool.t.sol

```diff
@@ -1433,11 +1433,14 @@ contract BPool_Unit_JoinswapPoolAmountOut is BasePoolTest {
     bPool.joinswapPoolAmountOut(tokenIn, _fuzz.poolAmountOut, _maxAmountIn);
   }
 
-  function test_Revert_NotFinalized() public {
+  function test_Revert_NotFinalized(
+    JoinswapPoolAmountOut_FuzzScenario memory _fuzz,
+    uint256 _maxAmountIn
+  ) public happyPath(_fuzz) {
     _setFinalize(false);
 
     vm.expectRevert('ERR_NOT_FINALIZED');
-    bPool.joinswapPoolAmountOut(tokenIn, 0, 0);
+    bPool.joinswapPoolAmountOut(tokenIn, _fuzz.poolAmountOut, _maxAmountIn);
```

Comment:
Could you fuzz all the function's general parameters within the `JoinswapPoolAmountOut_FuzzScenario` struct?